chromium_src/net/http/transport_security_state.cc

```diff
@@ -21,4 +22,118 @@
 
 #endif
 
+#define BRAVE_TRANSPORT_SECURITY_STATE_DELETE_DYNAMIC_DATA_FOR_HOST \
```

Comment:
can we add unit tests for DeleteDynamicDataForHost to cover this?

Reply:
added.